modules/core/src/main/scala/io/chrisdavenport/fuuid/FUUID.scala

```diff
@@ -5,7 +5,7 @@ import cats.implicits._
 import cats.effect.Sync
 import java.util.UUID
 
-final class FUUID private (private val uuid: UUID){
+final class FUUID private (private[fuuid] val uuid: UUID){
```

Comment:
Any thoughts on making this package private for integration ease? Or instead opt for an Unsafe function which can gain access to this value?